paasta_tools/frameworks/native_scheduler.py

```diff
@@ -150,12 +156,20 @@ def launch_tasks_for_offers(self, driver, offers):
                         driver.acceptOffers([offer.id], [operation])
 
                         for task in tasks:
+                            startup_timer = self.startup_timer_for_task(
```

Comment:
it's a nitpick but naming inconsistencies annoy me, we currently have: staging_timeout, start_timer and startup_timer. can we settle on one name? staging_timer / staging_timeout? startup_timer / startup_timeout?